internal/audit/auditlogtextformatter.go

```diff
@@ -179,6 +182,10 @@ func (f AuditLogTextFormatter) Format(entry *Entry) ([]byte, error) {
 		t.AppendRow([]interface{}{"Audited Dependencies", strconv.Itoa(packageCount)})
 		t.AppendSeparator()
 		t.AppendRow([]interface{}{"Vulnerable Dependencies", au.Bold(au.Red(strconv.Itoa(numVulnerable)))})
+    if numExcluded > 0 {
```

Comment:
super minor: formatting went wonky here. oh wait, who was using tabs previously!?!? Let the holy wars begin.